notebooks/examples/2 _Sentiment140.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
Traceback```sh
2020-08-24 11:13:17.163 | INFO     | subtest.multi_partner_learning:compute_test_score:200 - ### Evaluating model on test data:

---------------------------------------------------------------------------
InvalidArgumentError                      Traceback (most recent call last)
<ipython-input-12-3088d0d7b417> in <module>
----> 1 run_scenario(current_scenario)

~/.local/lib/python3.8/site-packages/subtest/scenario.py in run_scenario(current_scenario)
    715         is_save_data=True,
    716     )
--> 717     current_scenario.mpl.compute_test_score()
    718 
    719     # ----------------------------------------------------------

~/.local/lib/python3.8/site-packages/subtest/multi_partner_learning.py in compute_test_score(self)
    199         # After last epoch or if early stopping was triggered, evaluate model on the global testset
    200         logger.info("### Evaluating model on test data:")
--> 201         model_evaluation = model_to_evaluate.evaluate(
    202             x_test, y_test, batch_size=constants.DEFAULT_BATCH_SIZE, verbose=0)
    203         logger.info(f"   Model metrics names: {model_to_evaluate.metrics_names}")

~/.local/lib/python3.8/site-packages/keras/engine/training.py in evaluate(self, x, y, batch_size, verbose, sample_weight, steps, callbacks, max_queue_size, workers, use_multiprocessing)
   1355         self._make_test_function()
   1356         f = self.test_function
-> 1357         return training_arrays.test_loop(self, f, ins,
   1358                                          batch_size=batch_size,
   1359                                          verbose=verbose,

~/.local/lib/python3.8/site-packages/keras/engine/training_arrays.py in test_loop(model, f, ins, batch_size, verbose, steps, callbacks)
    447             batch_logs = {'batch': batch_index, 'size': len(batch_ids)}
    448             callbacks._call_batch_hook('test', 'begin', batch_index, batch_logs)
--> 449             batch_outs = f(ins_batch)
    450             if isinstance(batch_outs, list):
    451                 if batch_index == 0:

~/.local/lib/python3.8/site-packages/tensorflow/python/keras/backend.py in __call__(self, inputs)
   3790         value = math_ops.cast(value, tensor.dtype)
   3791       converted_inputs.append(value)
-> 3792     outputs = self._graph_fn(*converted_inputs)
   3793 
   3794     # EagerTensor.numpy() will often make a copy to ensure memory safety.

~/.local/lib/python3.8/site-packages/tensorflow/python/eager/function.py in __call__(self, *args, **kwargs)
   1603       TypeError: For invalid positional/keyword argument combinations.
   1604     """
-> 1605     return self._call_impl(args, kwargs)
   1606 
   1607   def _call_impl(self, args, kwargs, cancellation_manager=None):

~/.local/lib/python3.8/site-packages/tensorflow/python/eager/function.py in _call_impl(self, args, kwargs, cancellation_manager)
   1643       raise TypeError("Keyword arguments {} unknown. Expected {}.".format(
   1644           list(kwargs.keys()), list(self._arg_keywords)))
-> 1645     return self._call_flat(args, self.captured_inputs, cancellation_manager)
   1646 
   1647   def _filtered_call(self, args, kwargs):

~/.local/lib/python3.8/site-packages/tensorflow/python/eager/function.py in _call_flat(self, args, captured_inputs, cancellation_manager)
   1743         and executing_eagerly):
   1744       # No tape is watching; skip to running the function.
-> 1745       return self._build_call_outputs(self._inference_function.call(
   1746           ctx, args, cancellation_manager=cancellation_manager))
   1747     forward_backward = self._select_forward_and_backward_functions(

~/.local/lib/python3.8/site-packages/tensorflow/python/eager/function.py in call(self, ctx, args, cancellation_manager)
    591       with _InterpolateFunctionError(self):
    592         if cancellation_manager is None:
--> 593           outputs = execute.execute(
    594               str(self.signature.name),
    595               num_outputs=self._num_outputs,

~/.local/lib/python3.8/site-packages/tensorflow/python/eager/execute.py in quick_execute(op_name, num_outputs, inputs, attrs, ctx, name)
     57   try:
     58     ctx.ensure_initialized()
---> 59     tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,
     60                                         inputs, attrs, num_outputs)
     61   except core._NotOkStatusException as e:

InvalidArgumentError:  indices[74,61] = 15200 is not in [0, 15200)
	 [[node layer_embedding_9/embedding_lookup (defined at /home/zen/.local/lib/python3.8/site-packages/keras/backend/tensorflow_backend.py:3007) ]] [Op:__inference_keras_scratch_graph_504979]

Function call stack:
keras_scratch_graph

```

Reply:
I couldn't replicate the bug on this notebook
https://colab.research.google.com/drive/1dMHqvRHAj44ebpz7bbQ6NDyrCTCXW-EH?usp=sharing
Not entirely sure what is happening and whether or not I should investigate further.